scripts/build/build/targets.py

```diff
@@ -206,7 +206,20 @@ def BuildEfr32Target():
     target.AppendModifier('rpc', enable_rpcs=True)
     target.AppendModifier('with-ota-requestor', enable_ota_requestor=True)
     target.AppendModifier('sed', enable_sed=True)
-    target.AppendModifier('low-power', enable_low_power=True)
+    target.AppendModifier('low-power', enable_low_power=True).OnlyIfRe('-sed')
+    target.AppendModifier('chip_build_libshell', chip_build_libshell=True)
+    target.AppendModifier('disable_chip_logging', chip_logging=False)
+    target.AppendModifier('disable_chip_openthread_ftd', chip_openthread_ftd=False)
```

Comment:
I would remove enable/disable from naming to make it simpler. `no_ftd` ? or does this mean mtd so should it be `mtd` ?

Reply:
flag renamed to openthread_mtd, openthread was kept for more clarity.